docs/ThemeChooser.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react'
+import CozyTheme from '../react/CozyTheme'
+
+const ThemeChooser = ({ children }) => {
+  const [theme, setTheme] = useState('normal')
+  const handleChange = ev => setTheme(ev.target.value)
+  return (
+    <>
+      theme:{' '}
```

Comment:
Why do we need the `{' '}`?

Reply:
because there is a space after colons in English ? And it's the way to do this in JSX when a space precedes a component.